docs/app/pages/components/components-sections/page-header/page-header/page-header.component.html

```diff
@@ -50,6 +50,9 @@
     <tr uxd-api-property name="backClick" type="EventEmitter">
         The event fired when the back button is clicked.
     </tr>
+    <tr uxd-api-property name="logoClick" type="EventEmitter">
```

Comment:
We have recently changed how we document Outputs, this was never updated, but as all outputs are EventEmitters we now just specify the type of the event that is emitted, eg: `MouseEvent`

Reply:
Added <code>MouseEvent</code> to documentation